static/app/components/feedback/widget/feedbackModal.tsx

```diff
@@ -38,15 +39,18 @@ interface FeedbackFormData {
 async function sendFeedback(
   data: FeedbackFormData,
   pageUrl: string,
-  replayId?: string
+  replayId?: string,
+  type?: string
 ): Promise<Response | null> {
   const feedback = {
     message: data.comment,
     email: data.email,
+    name: data.name,
     replay_id: replayId,
     url: pageUrl,
   };
-  return await sendFeedbackRequest(feedback);
+  const tags = type ? {feedbackType: type} : null;
```

Comment:
@ryan953 Thoughts on this? A tag to have at least 1 level of categorization - e.g. feedback vs crons

Reply:
No strong feeling. It's not exposed in the list/search right now, so would it be useful to someone?

Reply:
It may, though it could be redundant due to URL (think about our case right now in saas app with feedback + crons)